src/cloudant/error.py

```diff
@@ -55,9 +56,14 @@ class CloudantArgumentError(CloudantException):
 
```

Comment:
The first paragraph of the description of this class needs updating as it specifies the old default `code` and talks about the message. I'd be inclined to remove the second sentence of the first paragraph entirely as it's repeating the information that's in the `param` annotations.